front/mailing/20231108-gdrive-issue.ts

```diff
@@ -0,0 +1,81 @@
+import sgMail from "@sendgrid/mail";
+
+import { front_sequelize } from "@app/lib/databases";
+
+const { SENDGRID_API_KEY = "", LIVE = false } = process.env;
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+export const sendAPIUserEmail = async ({
+  user_email,
+}: {
+  user_email: string;
+}) => {
+  const msg = {
+    to: user_email,
+    from: "spolu@dust.tt",
+    subject:
+      "[Dust] Resolved: Google Drive most recent documents not available for retrieval",
```

Comment:
I would say "Google Drive out of sync for 3h" ? please check final copy with @gabhubert :+1: